integration_tests/commands/async/hyperloglog_test.go

```diff
@@ -51,12 +51,6 @@ func TestHyperLogLogCommands(t *testing.T) {
 				"PFMERGE NON_EXISTING_SRC_KEY", "PFCOUNT NON_EXISTING_SRC_KEY"},
 			expected: []interface{}{"OK", int64(0)},
 		},
-		{
-			name: "PFMERGE with srcKey non-existing",
-			commands: []string{
-				"PFMERGE NON_EXISTING_SRC_KEY", "PFCOUNT NON_EXISTING_SRC_KEY"},
-			expected: []interface{}{"OK", int64(0)},
-		},
```

Comment:
Why is this removed?

Reply:
This test case is exactly same as the one above it. I guess it was copied by mistake.